client/miner.c

```diff
@@ -139,6 +159,89 @@ static int send_to_pool(struct xdag_field *fld, int nfld)
 	return 0;
 }
 
+static int miner_fresh_callback(void *data, xdag_hash_t hash, xdag_amount_t amount, xdag_time_t time, int n_our_key)
```

Comment:
data, amount, time and n_our_key  are unused

Reply:
Yes, they are required by xdag_traverse_our_blocks, but not used directly.

Reply:
So, maybe it makes sense to remove them?